compiler/src/dotty/tools/dotc/transform/patmat/Space.scala

```diff
@@ -641,6 +628,15 @@ class SpaceEngine(implicit ctx: Context) extends SpaceLogic {
 
   /** Display spaces */
   def show(s: Space): String = {
+    def params(tp: Type): List[Type] = {
```

Comment:
To clarify, I've checked that the length of `.firstParamTypes` does not actually change - all tests still pass if this function returns `ktor.fistParamTypes`.